setup.cfg

```
@@ -52,7 +52,7 @@ install_requires =
     packaging>=20.0
     pybind11>=2.5
     tabulate>=0.8
-    typing-extensions>=3.7
+    typing-extensions>=4.2
```

The recently released version of `typing-extensions 4.4` breaks `eve`, so for now we now to restrict the version range:
```suggestion
    typing-extensions>=4.2,<4.4
```

---

Thanks!